intellij_platform_sdk/BUILD.clion222

```diff
@@ -9,10 +9,10 @@ package(default_visibility = ["//visibility:public"])
 java_import(
     name = "sdk",
     jars = glob(
-        ["clion-*/lib/*.jar"],
```

Comment:
The tar file (CLion-222.2270.30.tar.gz) downloaded from the url for CLion 2022.2 EAP in WORKSPACE after commit https://github.com/bazelbuild/intellij/commit/d02e9491f6a847b405e45e1800d2535fcb03c945 has a folder called clion-{version} inside which the rest of the files are located. That is a difference between the file we get now and the one you were getting before (from a different url). So now we need to undo the changes you made to this file to make it work.